src/test/regress/sql/multi_extension.sql

```diff
@@ -207,6 +207,15 @@ SELECT * FROM print_extension_changes();
 ALTER EXTENSION citus UPDATE TO '9.4-1';
 SELECT * FROM print_extension_changes();
 
+-- Test upgrade paths for backported improvement of master_update_table_statistics function
+ALTER EXTENSION citus UPDATE TO '9.4-3';
+ALTER EXTENSION citus UPDATE TO '9.4-2';
+-- Should be empty result
+SELECT * FROM print_extension_changes();
+
+ALTER EXTENSION citus UPDATE TO '9.4-3';
+SELECT * FROM print_extension_changes();
+
```

Comment:
I think we should add 9.4-1 here as the upgrade will go from 9.4-1 to 9.5-1.

Reply:
I do not think that `release-9.4` branch needs to care about `9.5-1`. However your point may be valid on release branches for more recent major versions.